packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -2673,6 +2661,18 @@ export const components = ({
             },
           },
 
+          [`.${tableBodyClasses.root} &`]: {
```

Comment:
Seems like we should be changing the amount of padding in the cells when changing density. Rather than adjusting the height

Reply:
For table rows, I'd rather use heights (though I'm switching it to `minHeight` to avoid cutting off text that wraps onto multiple lines).

The height should be this value when the cell content is text, but it could be things besides text — checkbox, status chip, etc. I don't want us to need to recalculate the target height based on the content of the row.

Reply:
It really shouldn't matter what the content in the row is right? If it's a tight density, we make the paddingBlock smaller, looser density, we make it larger. Whenever we start manipulating heights explicitly, we run the risk of cutting off text

Reply:
I know using min-height here will account for that. Just not sure on the reasoning for height vs padding I guess. That is what padding is meant for